src/main/java/mate.academy/service/impl/ProductParserImpl.java

```diff
@@ -1,20 +1,24 @@
 package mate.academy.service.impl;
 
 import java.math.BigDecimal;
+import mate.academy.lib.Component;
+import mate.academy.lib.Inject;
 import mate.academy.model.Product;
 import mate.academy.service.ProductParser;
 
+@Component
 public class ProductParserImpl implements ProductParser {
     public static final int ID_POSITION = 0;
     public static final int NAME_POSITION = 1;
     public static final int CATEGORY_POSITION = 2;
     public static final int DESCRIPTION_POSITION = 3;
     public static final int PRICE_POSITION = 4;
+    @Inject
+    private Product product;
```

Comment:
Injector is aimed to substitute interface with it's implementation.
Product is a pure model which doesn't implement any interface so cannot and shouldn't be injected